Autopilot/Path Manager/PathManager.c

```diff
@@ -667,12 +667,37 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+// check if power on reset, brown out reset, watchdog time out or software reset flags are set
+
+ char checkForReset(){
```

Comment:
Either rename the function to something more obvious, or put a comment saying what the return value represents